it/main/PostgresTest.java

```diff
@@ -84,26 +89,29 @@ private void loadSomeData(EntityManager em, String filename) throws Exception {
         File inputFile = new File(getClass().getResource(filename).getPath());
 
         // Raw JDBC is significantly faster than EclipseLink for bulk loading data, so unwrap the connection
-        final Connection connection = em.unwrap(Connection.class);
-        final Statement statement = connection.createStatement();
-        final AtomicLong rows = new AtomicLong();
-        final long batchSize = 5000L;
-
-        Files.lines(inputFile.toPath()).forEach(line -> {
+        em.unwrap(Session.class).doWork(connection -> {
```

Comment:
If we're using Raw JDBC now anyway, do we still need to `unwrap` the connection?